src/pytorch_lightning/strategies/bagua.py

```diff
@@ -20,10 +20,12 @@
 from pytorch_lightning.trainer.states import TrainerFn
 from pytorch_lightning.utilities.distributed import ReduceOp
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.imports import _BAGUA_AVAILABLE
+from pytorch_lightning.utilities.imports import _package_available
 from pytorch_lightning.utilities.optimizer import optimizers_to_device
 from pytorch_lightning.utilities.seed import reset_seed
 
+_BAGUA_AVAILABLE = _package_available("bagua")
```

Comment:
Why it is required ?

Reply:
Because the class is strangely defined in if statement and actual meta pkg creation canot resolve it, BUT it is also the same way we already set for Deepspeed so it is consistent